odxtools/database.py

```diff
@@ -60,22 +60,21 @@ def __init__(self,
         comparam_specs: List[ComparamSpec] = []
         for root in documents:
             # ODX spec version
-            model_version = version(root.attrib.get("MODEL-VERSION", "2.0"))
             dlc = root.find("DIAG-LAYER-CONTAINER")
             if dlc is not None:
                 dlcs.append(DiagLayerContainer.from_et(dlc, []))
-            # In ODX 2.0 there was only COMPARAM-SPEC
-            # In ODX 2.2 content of COMPARAM-SPEC was renamed to COMPARAM-SUBSET
-            # and COMPARAM-SPEC becomes a container for PROT-STACKS
-            # and a PROT-STACK references a list of COMPARAM-SUBSET
-            if model_version >= version("2.2"):
```

Comment:
The test on the model version is required, otherwise the wrong parser will be used

Reply:
why? since COMPARAM-SUBSET is not present in ODX 2.0, the first `if` branch should never be taken?

(maybe COMPARAM-SPEC must be made aware of ODX 2.0, though...)

Reply:
> maybe COMPARAM-SPEC must be made aware of ODX 2.0, though...

Exactly.

Reply:
ok, fixed: 4985121279460073e65d8c931f05d890f75540ad

(Leaving that `if` alone would not have changed anything, though...)